setup.py

```diff
@@ -14,13 +14,9 @@
 
 install_requires = [
     "django>=3,<4",
-    # Todo: determine the versioning strategy PyFCM is using and adjust this if necessary.
-    # The last backward incompatible change seems to have been made in a minor version
-    # (1.4.0) so for now we'll assume we're safe within a minor version - and both 1.4.x
-    # and 1.5.x currently look good.
-    # https://github.com/olucurious/PyFCM/issues/285
-    "pyfcm>=1.4,<1.6",
+    "pyfcm==2.0.6",
     "django-konst>=2,<3",
+    "google-auth==2.22.0", # for loading credentials.
```

Comment:
pyfcm has this as a dependency, so we can rely on that

Reply:
I don't think it's good practice to rely on a package dependency to do things outside that package. Since we're doing the loading of google credentials outside PyFCM I think `django-fcm-devices` should have that dependency as well